lib/context.js

```diff
@@ -72,13 +82,13 @@ async function loadRuntimeConfig () {
   return context
 }
 
-export function storeRuntimeConfig () {
+export async function storeRuntimeConfig () {
```

Comment:
Did you check if there are any `storeRuntimeConfig()` calls which do not use await?

Reply:
yep, they all use await. I just had to add one in a test that was broken